packages/app/src/app/components/CreateSandbox/Import/AccountSelect.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import styled from 'styled-components';
+import VisuallyHidden from '@reach/visually-hidden';
+import {
+  useMenuState,
+  Menu,
+  MenuItem,
+  MenuDisclosure,
+  MenuSeparator,
+} from 'reakit/Menu';
+
+import { Stack, Text, Icon, Element } from '@codesandbox/components';
+import { useActions } from 'app/overmind';
+
+// In the new codebase this should be a Radix dropdown menu
+// https://www.radix-ui.com/docs/primitives/components/dropdown-menu
+// also, more composable if it were a design system component
```

Comment:
Could you use the [component](https://psp5z7-38799.preview.csb.app/?path=/story/components-menu--default-open) that exists in the shared components? Searched in the codebase and this is the only instance using `reakit/menu`:


![](https://user-images.githubusercontent.com/24959348/219975412-8d179331-0dfd-4281-bd3d-68b98d7c229d.png)

Might be easier to refactor when the codebase gets merged.

Reply:
I was looking at both and the Reach menu was missing a few things for me, like titles between items and horizontal splitters. I was tempted to use the Reach menu (since we were already using it) but implementing the missing functionality would have taken some more time. Both solutions aren't really perfect in this situation and I think when we want to move this to the new codebase, we should definitely use Radix alternatives.